### PR TITLE
Rename package in data bindings

### LINKED ISSUE #5: refactor package names
rename packages to represent new location

### LINKED ISSUE #7: Rename package in data bindings
### To Do
- [x] Rename `net.janrupf` to `net.labymedia`.
- [x] Move `net/janrupf/*` to `net/labymedia`.

### PR DESCRIPTION
**Pull request by xiantiel**
Closes #7

### CODE REVIEW

File: ultralight-java-databind/src/main/java/net/labymedia/ultralight/containers/JavascriptFunction.java [hunk 1]
```diff
@@ -16,6 +16,11 @@ public JavascriptFunction(final JavascriptClass functionClass) {
         this.functionClass = functionClass;
     }
 
+    public JavascriptFunction(final JavascriptClass functionClass, MethodHandle methodHandle) {
+        //TODO: do something with method handle
+        this.functionClass = functionClass;
+    }
+
```
Comment (PublicByte) [inline ultralight-java-databind/src/main/java/net/labymedia/ultralight/containers/JavascriptFunction.java:23]: Unrelated change.

File: ultralight-java-databind/src/main/java/net/labymedia/ultralight/containers/JavascriptFunction.java [hunk 2]
```diff
@@ -27,4 +32,9 @@ public void addMethodHandle(final MethodHandle method) {
             }
         }
     }
+
+    public Object methodHandle() {
+        //todo implement
+        return new Object();
+    }
```
Comment (PublicByte) [inline ultralight-java-databind/src/main/java/net/labymedia/ultralight/containers/JavascriptFunction.java:39]: Unrelated change.

**Review by PublicByte — CHANGES_REQUESTED** [review]
This PR contains unrelated changes ([add dummys commit](https://github.com/LabyMod/ultralight-java/pull/8/commits/14a714041560daa16a248a5b188ccdfeef4f37f6)), such as creating dummy methods. Please submit a separate PR for these changes (or none at all because they are dummy methods and do nothing).

**Comment by PublicByte** [comment]
Also, this PR refers to a closed issue. Consider creating a new issue or open said issue.

**Comment by Janrupf** [comment]
Has been done in place.

**Pull request closed without merging**